Makefile

```diff
@@ -81,12 +81,24 @@ simpy_cpython:
 		--exclude "*/bad*" \
 		--exclude "*/lib2to3/tests/data/*"
 
+cpython:
+	tar xf data/cpython-lib.tgz
+
+no-cpython:
```

Comment:
Nit (feel free to ignore): Maybe `clean-cpython`? The reason is that is not clear that depending on `no-cpython` will remove things.

Reply:
Done.